core/domain/learner_progress_services.py

```diff
@@ -21,6 +21,7 @@
 from core.domain import subscription_services
 from core.domain import user_domain
 from core.platform import models
+from google.appengine.ext import ndb
```

Comment:
I didn't really understand this line. Where exactly is core.platform.datastore - are you referring to one of the gae_model files? I do need ndb to use ndb.get_multi to fetch all the summaries for all the entities.